app/assets/stylesheets/application/donations/donations.scss

```diff
@@ -40,9 +40,34 @@
   width: 32%;
 }
 
+label[for="donation_amount_other"] {
+  margin-top: 1px;
+  vertical-align: middle;
+  width: 32%;
+  #donation_amount_other {
```

Comment:
Rule declaration should be preceded by an empty line<br>Avoid using id selectors<br>Selector `donation_amount_other` should be written in lowercase with hyphens